### ISSUE TITLE
Strange Bracken results indicating Brassica Rapa reads in the Human RNASeq COVID Samples

### ISSUE THREAD

**Issue by Rohit-Satyam** [jenniferlu717/Bracken#243]
Dear @jenniferlu717 

I have been using Bracken for a long time but recently something changed. I have clinical COVID RNAseq samples (380) and I realized that in many samples Bracken shows the abundance of _Brassica Rapa_ and other Plant-related reads such as _Juglans regia_. This is completely opposed to what we observe in Kraken2 results. Besides, these are human samples so I shouldn't observe Viridiplantae reads. I see that a similar misassignment has also been reported 

`Version Used: Bracken v2.9`

```
kraken2 --threads 12 --confidence 0.1 --paired --db PlusPF_database_indexes/ \
 file.R1.fastq.gz file.R1.fastq.gz --output $file --report-minimizer-data \
  --report file_kraken.report --gzip-compressed --unclassified-out file_unclassified#.fasta
bracken -r 100 -d <path> -i file_kraken.report -o file_abundance.txt -w file_bracken.report
```

![image](https://github.com/jenniferlu717/Bracken/assets/55479480/355a2a49-08fe-4583-aaf0-b7e47d54b0f6)

Also attaching the results:

[bracken_misassignment.zip](https://github.com/jenniferlu717/Bracken/files/13830431/bracken_misassignment.zip)

**Comment by jenniferlu717** [comment]
I looked back into the code and what Bracken is trying to do is redistribute reads based on the kmer distribution file. 

So for example, if any sequence (100bp for example) from Brassica Rapa is classified as Bacteria or Acinetobacter, then reads from those levels were redistributed to Brassica Rapa.

Are you using a complete genome of Brassica Rapa? 

What I could do is require that any redistribution has to remain in the same tree. This normally doesn't cause problems unless genomes are contaminated.

**Comment by Rohit-Satyam** [comment]
Hi @jenniferlu717. My data is RNAseq (not genomes) but the read length is greater than 100 post trimming. All I am using are Kraken2/ Bracken index PlusPF from [here](https://benlangmead.github.io/aws-indexes/k2).

>What I could do is require that any redistribution has to remain in the same tree. This normally doesn't cause problems unless genomes are contaminated.

Yep, that would be great. In addition, can you tell why there is missing taxa for the Bacteria in the Sankey plot? I observe that such a discrepancy has been  reported in Pavian (see [here](https://github.com/fbreitwieser/pavian/issues/76#issuecomment-1877132352) and [here](https://github.com/fbreitwieser/pavian/issues/35)) but this must be stemming from the Kraken2 reports right?

**Comment by Rohit-Satyam** [comment]
Hi, I was wondering if this is a quick fix and can be inculcated by the end of this week or if I have to wait for a few months. kindly let me know because I have a presentation coming. 

Thanks

**Comment by Rohit-Satyam** [comment]
Hi @jenniferlu717 I was wondering if this issue was fixed.

**Comment by Rohit-Satyam** [comment]
I don't think this issue has yet been fixed. I was using bracken again, and I see reads being assigned to plants in clinical sample
